pcl_ros/include/pcl_ros/pcl_node.hpp

```diff
@@ -208,12 +270,14 @@ class PCLNodelet : public nodelet_topic_tools::NodeletLazy
   inline bool
   isValid(const ModelCoefficientsConstPtr & model, const std::string & topic_name = "model")
   {
+    // TODO(daisuke): should I leave this commented out?
```

Comment:
For both TODOs of this nature, can you check the git commit that commented them out and see why they did that? If you go to this file and do "History" tab, you can see the history of the file (or if you do "Blame" you can see the last to touch those lines)

Reply:
Both have been commented out since the initial commit. It might be commented out before migrating to GitHub.

> migrating nodelets and tools from fuerte release to pcl_ros
https://github.com/ros-perception/perception_pcl/blame/0ac68106885f170ab5ba30d7ba96e6ba5a048253/pcl_ros/include/pcl_ros/pcl_nodelet.hpp#L175-L179

Reply:
I'd leave it alone then. Remove TODO